pkg/util/logging/default_logger.go

```diff
@@ -32,10 +32,14 @@ func DefaultHooks() []logrus.Hook {
 }
 
 // DefaultLogger returns a Logger with the default properties
-// and hooks.
-func DefaultLogger(level logrus.Level) *logrus.Logger {
+// and hooks. The desired output format is passed as a string.
+func DefaultLogger(level logrus.Level, format string) *logrus.Logger {
```

Comment:
I'd prefer to define an enum in this package for the valid formats and use that everywhere rather than a plain string.